0001-Enable-pbuffer-to-fix-most-deqp-pbuffer-testcases.patch

```diff
@@ -0,0 +1,284 @@
+From c666b63572c8d4f116fe7d10c57a20092a74c0c6 Mon Sep 17 00:00:00 2001
+From: Liu Zhiquan <zhiquan.liu@intel.com>
+Date: Mon, 7 Nov 2016 15:32:17 +0800
+Subject: [PATCH] Enable pbuffer to fix most deqp pbuffer testcases.
+
+mesa android path didn't support pbuffer, so add pbuffer
+support to fix most deqp and cts pbuffer test cases fail;
+add support front buffer and single buffer config;
+
+Bug ID: https://github.com/android-ia/device-androidia/issues/6
```

Comment:
ok I believe I spotted everything, please clean up those and then send to mesa-dev!